proxy/io.go

```diff
@@ -25,45 +23,35 @@ func WriteChunks(conn *net.TCPConn, c [][]byte) (n int, err error) {
 	return total, nil
 }
 
-func ReadBytes(conn *net.TCPConn) ([]byte, error) {
-	ret := make([]byte, 0)
-	buf := make([]byte, BUF_SIZE)
-
-	for {
```

Comment:
Previously, this read loop would run once, except when the entire buffer was read. Decided to remove the unnecessary loop.